### PR TITLE
first draft build local multi-arch images

### PR DESCRIPTION
**Pull request by tballison**

### CODE REVIEW

**Comment by stumpylog** [comment]
This works for me, except for the pushing obviously.  One minor annoyance, if the build does fail, the builder will still be around, so it will have to be manually removed before running again.  Minor, but something to remember

**Comment by tballison** [comment]
Got it. Thank you @stumpylog !

How's this: https://hub.docker.com/layers/apache/tika/2.9.2-alpha-multi-arch/images/sha256-b8b6e02e3e9f98ddae33b74881f4ead7846ee12352d53149098857378bb3393d?context=repo

**Comment by tballison** [comment]
Building `full` now...

**Comment by stumpylog** [comment]
I pulled the image on a Pi (arm64) and had no problems pulling or starting.

File: docker-tool.sh [hunk 1]
```diff
@@ -17,15 +17,24 @@
 #   specific language governing permissions and limitations
 #   under the License.
 
+stop_and_die() {
+  docker buildx stop tika-builder || die "couldn't stop builder -- make sure to stop the builder manually! "
```
Comment (stumpylog) [inline docker-tool.sh]: I'd make this `docker buildx rm` instead.  Just stopping it seems to have the same effect, complaints about the existing instance

**Comment by stumpylog** [comment]
Seems good to me.  I built it a couple times, and the recent updates makes the builder clean up better

**Pull request merged**